specification/metrics/purefb-metrics.md

```diff
@@ -62,9 +62,15 @@ This document describes the semantic conventions for Pure FlashBlade Metrics.
 | --------- | ------------------ | ---------------------------- | ----- | ----------------------------------------------------------------------------------------------------------------------- | ---------- | ---------------- | ----------------------------- |
-| Available | purefb_alerts_open | FlashBlade open alert events |       | Gauge                                                                                                                   | Double     | `component_name` | (component name)              |
+| Available | purefb_alerts_open | FlashBlade open alert events |       | Gauge                                                                                                                   | Double     | `name`           | (name)                        |
+|           |                    |                              |       |                                                                                                                         |            | `action`         | (action)                      |
+|           |                    |                              |       | The code number of the event                                                                                            |            | `code`           | (code)                        |
+|           |                    |                              |       |                                                                                                                         |            | `component_name` | (component name)              |
 |           |                    |                              |       |                                                                                                                         |            | `component_type` | (component type)              |
+|           |                    |                              |       | The time the alert was created in milliseconds since the UNIX epoch                                                     |            | `created`        | (created)                     |
+|           |                    |                              |       | Knowledge Base URL related to the alert                                                                                 |            | `kburl`          | (kburl)                       |
 |           |                    |                              |       |                                                                                                                         |            | `severity`       | `info`, `warning`, `critical` |
+|           |                    |                              |       | A summary of the alert                                                                                                  |            | `summary`        | (summary)                     |
```

Comment:
summary was deleted from the code, and should be deleted here too
```suggestion
```

Reply:
`summary` was never removed. It was `description`